src/Symfony/Component/Notifier/Bridge/Firebase/README.md

```diff
@@ -1,14 +1,13 @@
 Firebase Notifier
 =================
 
-Provides Firebase integration for Symfony Notifier.
+Provides [Firebase](https://firebase.google.com) integration for Symfony Notifier.
 
 DSN example
 -----------
 
-```
-// .env file
-FIREBASE_DSN=firebase://USERNAME:PASSWORD@default
+```ini
+firebase://USERNAME:PASSWORD@default
```

Comment:
I would keep the whole thing as knowing which env var you need to set (as used by the recipe is important as well.)

Reply:
Ok I will add them back and add the ones which ate missing 👌🏻

Reply:
Done